src/main/com.tgt.testapp/database/PostgresDAO.java

```diff
@@ -11,18 +11,57 @@
     public static void main(String[] args) {
 
         String url = "jdbc:postgresql://localhost:5432/postgres";
+        String userName = "postgres";
+        String password = "Sadhguru";
 
-        try (Connection con = DriverManager.getConnection(url);
-             Statement st = con.createStatement();
-             ResultSet rs = st.executeQuery("SELECT * from user_cred_table")) {
+        String insertionQuery1 = "INSERT INTO public.user_cred_table\n" +
```

Comment:
can you make this more user friendly? For example - let the user choose what operation they want to do and then you do the SQL operation.